lib/child_process.js

```diff
@@ -195,6 +195,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
 
   var ex = null;
 
+  var cmd = file;
+
```

Comment:
@ChALkeR There was a bit different case in the sloppy mode: r[eassigning a defined parameter + mentioning `arguments`](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#31-reassigning-a-defined-parameter-while-also-mentioning-arguments-in-the-body-in-sloppy-mode-only-typical-example).